src/main/java/database/Storage.java

```diff
@@ -0,0 +1,12 @@
+package database;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static Map<String, Integer> assortment = new HashMap<>();
```

Comment:
According to the encapsulation principle, you should hide the inner class elements with the help of access modifiers. Make `assortment` private and provide public methods to interact with it.